lib/php/rendition-factory/src/Transformer/TransformerConfigHelper.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Alchemy\RenditionFactory\Transformer;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class TransformerConfigHelper
```

Comment:
I don't think this class is used.
A Helper class should be readonly and contain the `createBaseTree` method only. The rest could stay on the Document class which has a hierarchy.